openedx.yaml

```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: andr
+oeps: {}
+owner: edx/edx-mobile-push
+tags: [mobile]
```

Comment:
Yeah, definitely. Feel free to edit the PR.

The syntax would be something like:
```
tags:
    - mobile
    - android
```